### ISSUE TITLE
Warning when [aria-hidden="true"] elements contain focusable descendants

### ISSUE THREAD

**Issue by SavvasStephanides** [jackdomleo7/Checka11y.css#97]
### Describe the new a11y feature or project enhancement

Whenever an element has `aria-hidden="true"`, it shouldn't have descendants that are focusable. These are:

* `<a>`
* `<area>`
* `<audio controls>`
* `<button>`
* `<iframe>`
* `<input>`
* `<select>`
* `<summary>`
* `<textarea>`
* `<video controls>`
* Any element with the contentEditable attribute
* Any element with a `tabindex` set to a numeric value other than `-1`

Example

```html
<div aria-hidden="true">
    <button>Click me!</button> <!-- <-- This should show a warning. -->
</div>
```

### Describe the solution you'd like

The solution would be to display a warning whenever a focusing element is present:

Example:

```scss
[aria-hidden="true"]{
    button::after{
        @include contentMessage(warning, "Focusable element in a hidden element.");
	}
}
```

#### ⚠ The problem

The problem is that some elements such as `<select>` can't have a `:after` class, so it won't work. Maybe there's a way to style the parent element instead?

### Link(s)

[[aria-hidden="true"] elements contain focusable descendants - Google Developers Web.Dev](https://web.dev/aria-hidden-focus/)

**Comment by jackdomleo7** [comment]
Good idea! We've accounted for elements that can't have `::after`: In `src/shared/messages/_vector.scss`, you can find a mixin for applying an SVG background image. You can find an example of this when an image doesn't have any `alt` text.
